Examples/ObjectiveC/LabelPlacementExample.m

```diff
@@ -0,0 +1,40 @@
+#import "LabelPlacementExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleLabelPlacement = @"LabelPlacementExample";
+
+@interface LabelPlacementExample ()<MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation LabelPlacementExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.mapView = [[MGLMapView alloc] initWithFrame: CGRectMake(0, 0, self.view.bounds.size.width, self.view.bounds.size.height)];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    // Set the map's initial style, center coordinate, and zoom level
+    self.mapView.styleURL = [MGLStyle streetsStyleURL];
+    [self.mapView setCenterCoordinate:CLLocationCoordinate2DMake(37.791282, -122.396301)
+                            zoomLevel:15.0
+                             animated:NO];
+    self.mapView.delegate = self;
+    [self.view addSubview:self.mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    MGLSymbolStyleLayer *poiLabelLayers = (MGLSymbolStyleLayer *)[self.mapView.style layerWithIdentifier:@"poi-label"];
```

Comment:
Its important to mention here that the `poi-label` layer is specific to Mapbox styles that include the `mapbox-streets-v8` source, which you may want to link to here: https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#poi_label